plugins/logs/plugin.go

```diff
@@ -583,12 +594,6 @@ func (p *Plugin) Log(ctx context.Context, decision *server.Info) error {
 		return proxy.Log(ctx, event)
```

Comment:
Kept this for backwards-compatibility: If the plugin fails, the `Log()` call fails.